src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/CheckboxesContainerComponent.tsx

```diff
@@ -65,138 +67,104 @@ const useStyles = makeStyles({
   margin: {
     marginBottom: '1.2rem',
   },
-});
-
-function usePrevious(value) {
-  const ref = React.useRef();
-  React.useEffect(() => {
-    ref.current = value.slice();
-  });
-
-  return ref.current;
-}
-
-export const CheckboxContainerComponent = (props: ICheckboxContainerProps) => {
-  const classes = useStyles(props);
-  const apiOptions = useAppSelector(state => state.optionState.options[props.optionsId]);
-  const options = apiOptions || props.options || [];
-  const [selected, setSelected] = React.useState([]);
-  const prevSelected: any = usePrevious(selected);
-  const checkBoxesIsRow: boolean = options.length <= 2;
+}));
+
+const defaultOptions: IOption[] = [];
+const defaultSelectedOptions: string[] = [];
+
+export const CheckboxContainerComponent = ({
+  id,
+  options,
+  optionsId,
+  formData,
+  preselectedOptionIndex,
+  handleDataChange,
+  handleFocusUpdate,
```

Comment:
This is not required when you're not handling focus manually any more.
```suggestion
```

Reply:
This also fires off an action, and while I'm not really sure what that purpose is either, I would prefer to keep this in just in case. We will do a more thorough investigation of that action, what it does, and if its really needed later. If its not needed, then all the usages will get cleaned up.

Reply:
Yea, the functionality and related redux state is only used in Checkbox and RadioButton, so it is totally safe to keep.